db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,81 @@
+class AddCommentsToVmsTable < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines and Templates"
+
+    change_column_comment :vms, :autostart, "Indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "ID of the availability zone that the VM is part of."
+    change_column_comment :vms, :busy, "[DEFINITION NEEDED]"
```

Comment:
I don't actually know what this one is and I can't find anyone who sets it, need to do more investigation on this one...

Reply:
@gtanzillo @Fryguy ?

Reply:
I want to say this is old (circa '07) SSA stuff.  It is should not be needed anymore.

Reply:
@agrare @gmcculloug Currently set to "Indicates whether or not the VM is being scanned for SmartState analysis. Deprecated.". If that's ok, please mark this resolved.

Reply:
I'm good with this one, but I don't have the "Resolve Conversation" button on this repo.  So please accept this in it's place: :white_check_mark: